core/src/dynamic.rs

```diff
@@ -8,7 +8,7 @@
 use crate::metadata::{DecodeWithMetadata, Metadata};
 use alloc::vec::Vec;
 use scale_decode::DecodeAsType;
-pub use scale_value::{At, Value};
+pub use scale_value::{scale, serde, stringify, At, Value};
```

Comment:
can't you guys use [subxt::ext::scale_value](https://github.com/paritytech/subxt/blob/master/subxt/src/lib.rs#L109) instead of relying of this re-export from core?

Reply:
I agree that it's better to use the ext imports (or depend on the same version of scale-value in Cargo.toml) instead of subxt exporting everything; lessens the burden on subxt if we don't have to re-export everything from dependent libraries :)

Reply:
Thanks, I wasn't aware of `subxt::ext`. That solves it.